### PR TITLE
Added Media attachment for QField

### PR DESCRIPTION
**Pull request by SeqLaz**
Added the media attachment with examples

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3113 Document multimedia attachments for QField](https://app.clickup.com/t/2192114/QF-3113)

File: documentation/how-to/attributes-form.en.md [hunk 1]
```diff
@@ -85,37 +85,55 @@ information (e.g. percentage) in the linking table.
...
 ![](../assets/images/hyperlink_option.png)
 
+## Configurable attachment path
+:material-monitor: Desktop preparation
+
+QFieldSync provides the possibility to configure the path of attachments.
+
+- Go to _QFieldSync plugin > Layers Properties_
+
+Use expressions to specify the path of the attachments. By default, pictures are saved into the "DCIM" folder, audio are saved into the "audio" folder and videos are saved into "video" with a timestamp as name.
```
Comment (m-kuhn) [inline documentation/how-to/attributes-form.en.md:133]:
```suggestion
Use expressions to specify the path of the attachments. By default, pictures are saved into the "DCIM" folder, audio recordings are saved into the "audio" folder and videos are saved into "video" with a timestamp as name.
```

**Comment by m-kuhn** [comment]
Very nice, some minimal suggestions left.

**Review by m-kuhn — APPROVED** [review]

**Comment by SeqLaz** [comment]
> @SeqLaz do you have the permissions to merge?

No I don't have the permissions, and also are other pending to merge pull requests, like the QR code

**Pull request merged**